docs/partners/2-certification/available-certifications/sis-v5/test-scenarios/staff-scenarios.md

```diff
@@ -0,0 +1,53 @@
+# v5 StaffAssociation > Staff Scenarios
+
+StaffAssociation defines staff information. It can be used to define employment,
+assignment, and teaching associations, as well as staff positions
+and staff leave events.
+
+The Staff represents an individual who performs specified activities for any
+public or private education institution or agency that provides instructional
+and/or support services to students or staff at the early childhood level
+through high school completion. For example, this includes:
+
+* An "employee" who performs services under the direction of the employing
+  institution or agency is compensated for such services by the employer and is
+  eligible for employee benefits and wage or salary tax withholdings.
+* A "contractor" or "consultant" who performs services for an agreed upon fee or
+  an employee of a management service contracted to work on site.
+* A "volunteer" who performs services on a voluntary and uncompensated basis.
+* An in-kind service provider.
+* An independent contractor or businessperson working at a school site.
+
+## Prerequisites
+
+* Before executing the PUT Statements, you must
+  create [StaffEducationOrganizationAssignmentAssociation](./staff-education-organization-assignment-association-scenarios.md)
```

Comment:
I think this is also because of the `.../sis/test-scenarios/...` to `.../sis-v5/test-scenarios/...` renaming thing.

Reply:
Yes... but I had used an approach that should have made it work. Worked in most of the renamed pages. Oh well.

(usually if you rename a file, then stage that rename, now you can edit, and stage the edit. Should show up as a rename with edit instead of as a new page).